modal/cli/run.py

```diff
@@ -136,20 +136,33 @@ def _get_clean_app_description(func_ref: str) -> str:
 
 
 def _get_click_command_for_function(app: App, function_tag):
-    function = app.registered_functions[function_tag]
+    function = app.registered_functions.get(function_tag, None)
```

Comment:
Super nit (not worth fixing) the `None` is superfluous here, that's the default behavior for `.get()`